### PR TITLE
OpenBrush follow-up delivery milestone 4

### PR DESCRIPTION
**Pull request by 0xMarkian**
# Milestone Delivery Checklist

- [x] The [milestone-delivery-template.md](https://github.com/w3f/Grant-Milestone-Delivery/blob/master/deliveries/milestone-delivery-template.md) has been copied and updated.
- [x] The [invoice form :pencil:](https://forms.gle/LSRr7PCjBpEbKGh89) has been filled out for this milestone.
- [x] This pull request is being made by the same account as the accepted application.
- [x] In case of acceptance, the payment will be transferred to the BTC/ETH payment address provided in the application.  
- [x] The delivery is according to the [Guidelines for Milestone Deliverables](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).

Link to the application pull request: https://github.com/w3f/Grants-Program/pull/621

We have changed the ERC20 address, here is the pull request for it - https://github.com/w3f/Grants-Program/pull/850

### CODE REVIEW

File: deliveries/openbrush_follow_up-milestone_2.md [hunk 1]
```diff
@@ -0,0 +1,30 @@
+# Milestone Delivery :mailbox:
+
+**The [invoice form :pencil:](https://docs.google.com/forms/d/e/1FAIpQLSfmNYaoCgrxyhzgoKQ0ynQvnNRoTmgApz9NrMp-hd8mhIiO0A/viewform) has been filled out correctly for this milestone and the delivery is according to the official [milestone delivery guidelines](https://github.com/w3f/Grants-Program/blob/master/docs/milestone-deliverables-guidelines.md).**  
+
+* **Application Document:** [OpenBrush follow-up](https://github.com/w3f/Grants-Program/pull/621).
```
Comment (alxs) [inline deliveries/openbrush_follow_up-milestone_2.md:5]:
```suggestion
* **Application Document:** [OpenBrush follow-up](https://github.com/w3f/Grants-Program/blob/master/applications/openbrush-follow-up.md).
```

**Review by alxs — CHANGES_REQUESTED** [review]

**Comment by alxs** [comment]
Thanks for the delivery @0xMarkian. We'll look into it as soon as possible.

**Comment by semuelle** [comment]
Hi @0xMarkian. I am reviewing your delivery and seem to be unable to compile it. I am getting a bunch of `Couldn't open temporary storage: No such file or directory` error messages, as shown below. Am I supposed to use a specific branch/commit or toolchain version? 

```sh
❯ yarn build
yarn run v1.22.15
$ npx redspot compile
compiler version: 0.17.0
toolchain: nightly
ink: 32 matches

===== Compile lending_contract =====

 [1/5] Building cargo project
    Updating git repository `https://github.com/paritytech/ink`
    Updating crates.io index
   Compiling contracts v1.4.0 (/openbrush/openbrush-contracts/contracts)
error: custom attribute panicked
  --> /openbrush/openbrush-contracts/contracts/traits/access_control/mod.rs:16:1
   |
16 | #[brush::trait_definition]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: Couldn't open temporary storage: No such file or directory (os error 2)

error: custom attribute panicked
  --> /openbrush/openbrush-contracts/contracts/traits/flashloan/mod.rs:16:1
   |
16 | #[brush::trait_definition]
   | ^^^^^^^^^^^^^^^^^^^^^^^^^^
   |
   = help: message: Couldn't open temporary storage: No such file or directory (os error 2)

...
```

**Comment by xgreenx** [comment]
Hi, thanks for the report! 
Some users also experience that issue in Ubuntu. The fix is already merged:
https://github.com/Supercolony-net/openbrush-contracts/pull/83

**Comment by TtomaS7** [comment]
Hey!
We are waiting for your review, @semuelle

**Comment by semuelle** [comment]
Hi @0xMarkian & @xgreenx. Thank you for the quick fix. I was able to compile and run all unit and integration tests without any further hiccups. Kudos to you for all the effort you have put into generalising the contracts and creating, implementing the standards and providing documentation.

Your milestone is hereby accepted. I will forward your invoice for processing.

**Comment by RouvenP** [comment]
Hi @0xMarkian @xgreenx @TtomaS7, we transferred the payment today. Thanks!

**Pull request merged**